### PR TITLE
Documented models for atmospheric scattering and absorption

### PR DESCRIPTION
**Pull request by nollety**
# Description

I've added a new section to the user guide to describe what models we use for molecular scattering and molecular absorption in the atmosphere.

# Checklist

- ~~[ ] The code follows the relevant coding guidelines~~
- ~~[ ] The code generates no new warnings~~
- ~~[ ] The code is appropriately documented~~
- ~~[ ] The code is tested to prove its function~~
- [x] The feature branch is rebased on the current state of the `main` branch
- [x] I give permission that the Eradiate project may redistribute my contributions under the terms of its license

### CODE REVIEW

File: docs/rst/user_guide/atmosphere/absorption.rst [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+.. math::
+
+   k_{\mathrm a \lambda} = n \, \sigma_{\mathrm a}(\nu, p, T)
+
+where
+:math:`n` is the air number density,
+:math:`\sigma` is the monochromatic absorption cross section,
+:math:`\nu = \lambda^{-1}` is the wavenumber,
+:math:`p` is the pressure and
+:math:`T` is the temperature.
```
Comment (leroyvn) [inline docs/rst/user_guide/atmosphere/absorption.rst:17]:
```suggestion
* :math:`n` is the air number density,
* :math:`\sigma` is the monochromatic absorption cross section,
* :math:`\nu = \lambda^{-1}` is the wavenumber,
* :math:`p` is the pressure and
* :math:`T` is the temperature.
```

File: docs/rst/user_guide/atmosphere/absorption.rst [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+   * - Mixing ratio [%]
+     - 78.084
+     - 20.9476
+     - 0.0314
+     - 0.0002
+
+The ``us76_u86_4`` is named in this way because it corresponds to the
+gas mixture defined by the ``us76`` atmosphere model
+:cite:`NASA1976USStandardAtmosphere` in the region of altitudes below 86 km,
+and restricted to the 4 main molecular species, namely N2, O2, CO2 and CH4.
```
Comment (leroyvn) [inline docs/rst/user_guide/atmosphere/absorption.rst:56]: This should go above the table: it describes what this profile is really about.

File: docs/rst/user_guide/intro.rst [hunk 3]
```diff
@@ -8,6 +8,7 @@ User guide
 
    basic_concepts
    onedim_solver_app
+   atmosphere/intro
    package_structure
    data_guide
    unit_guide_user
```
Comment (leroyvn) [inline docs/rst/user_guide/intro.rst:14]: Let's put modelling notes at the end after the content dedicated to using the software.

```suggestion
   basic_concepts
   onedim_solver_app
   package_structure
   data_guide
   unit_guide_user
   atmosphere/intro
```

File: docs/rst/user_guide/atmosphere/intro.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.. _sec-atmosphere-intro:
```
Comment (leroyvn) [inline docs/rst/user_guide/atmosphere/intro.rst:1]: I'd like to change conventions and use `index.rst` as the landing page for each directory.

**Review by leroyvn — APPROVED** [review]
Hi @nollety, thanks for this PR. I requested a few changes, among which one is to rename `intro.rst` to `index.rst` (convention used in websites and better suited in many cases in our docs because the "intro" page actually most often contains nothing.

**Comment by nollety** [comment]
@leroyvn I added your changes. I've also added the dimensions of the physical quantities that are introduced, so as to avoid any confusion, the normalisation rule that we use for the phase function, and other minor things.

**Comment by leroyvn** [comment]
Good! As usual: rebase + squash and we'll merge this!

**Comment by nollety** [comment]
@leroyvn it's ready to be merged :)

**Pull request merged**